src/main/webapp/script.js

```diff
@@ -10,4 +10,55 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(getChart);
```

Comment:
It should be noted that if we have multiple pages, the chart functionality may need to be moved to a seperate script. However, for now, this should be fine.